understanding/20/contrast-enhanced.html

```diff
@@ -73,13 +73,19 @@ <h2>Intent of Contrast (Enhanced)</h2>
       </div>
 
       <div class="note">
-         <p>Because authors do not have control over user settings for font smoothing/anti-aliasing, when evaluating this
-            Success Criterion, refer to the foreground and background colors obtained from the user agent, or the underlying
-            markup and stylesheets, rather than the text as presented on screen.</p>
-         <p>Due to anti-aliasing, particularly thin or unusual fonts may be rendered by user agents with a much fainter
-            color than the actual text color defined in the underlying CSS. This can lead to situations where text has
-            a contrast ratio that nominally passes the Success Criterion, but has a much lower contrast in practice.
-            In these cases, best practice would be for authors to choose a font with stronger/thicker lines,
+         <p>User agents which render text will often apply anti-aliasing, font smoothing, and other techniques
+            which can cause some pixels of text to render in fainter colors than those specified by authors.
+            When evaluating this Success Criterion, this is usually not relevant; as long as the applied font,
+            size, and weight ensure that some pixel of each character reaches the author-specified color, evaluators
+            may refer to the foreground and background colors obtained from the user agent, or the underlying
```

Comment:
"may refer" introduces inconsistency in evaluation. some authors/testers will, others won't. and for those that don't, and instead rely solely on pixel-peeping, will have further variation in their findings, because the actual rendered pixels on *their* display will depend on a myriad of different factors:

* their particular display's physical resolution (actual hardware pixels) and logical resolution (what their OS' resolution is) - compare "retina" and "non-retina" displays (paradoxically, this means a tester who uses two monitors - e.g. their laptop's built-in monitor and a nice 8k external monitor set to run at any resolution lower than pure 8k, like 1920x1080 - will get different results when pixel-peeping on one monitor versus the other monitor
* the antialiasing settings on their machine - for instance on Windows, there are various ClearType settings/modes available, which all subtly change the way antialiasing works
* the specific font present on the specific machine - for instance, there are many variations in the wild for even common fonts like Helvetica ... different foundries have subtly different variations, with different hinting/metrics, that can all end up making a difference in how text is rendered on a specific machine; beyond that, there is no guarantee that a particular author, user, tester has the specified font available, and the browser may start to do font substitutions (following the list of fallback fonts specified, which all have again the same issue); similarly, browsers/OSs may also "fake" certain variations, like bold or italic, if there is no explicit font installed on the system that has been designed explicitly as the bold/italic/etc variant
* pixel-peeping will - depending on the platform, and tool used to grab the pixel values - often be influenced by colour management/profiles on the specific machine, which will introduce subtle variations (which, particularly on colour combinations that are "just" on the verge of passing/failing, may make all the difference in shifting the values one way or another)

in short, this still handwaves (with the "may") how this binary pass/fail criterion is to be evaluated. while relying on the colour values as defined in the underlying technology (e.g. CSS) may end up not reflecting the real-world/rendered version as seen on a user's device, it at least provides a much higher likelihood of inter-tester consistency. yes, the underlying defined colour may not reflect the *actual* reality for users in all cases, but that's not new and should be addressed as a strong best practice recommendation instead (to close the gap between "conforms to the letter of the SC" and "is accessible in practice")